include/pmsis/drivers/gpio.h

```diff
@@ -39,39 +40,114 @@
 
 /**@{*/
 
+/**
+ * \brief Macro used to retrieve GPIO pin number.
+ *
+ * Right shift to do to get GPIO's pin number(needed to set the pin for GPIO mode).
+ */
+#define PI_GPIO_NUM_SHIFT  8
+/**
+ * \brief Macro used to retrieve GPIO number.
+ *
+ * Mask to apply to get GPIO's number.
+ */
+#define PI_GPIO_NUM_MASK   0xFF
```

Comment:
It is better to not expose these constants to the user, this should be kept internal, no ?